package.json

```diff
@@ -26,7 +26,7 @@
     "react-dom": "^16.13.1",
     "strapi-utils": "3.6.2",
     "styled-components": "^5.2.0",
-    "typescript": "^4.0.3"
+    "typescript": "4.4.2"
```

Comment:
pinned intentional, I assume

Reply:
Totally intentional.  The update to the latest version (4.4.2) caused issues with the plugin being installed because newer validation rules were failing thus causing the install to fail.  We'll just have to be more intentional going forward when we update the library.